test/db/analysis/rx

```diff
@@ -0,0 +1,367 @@
+|           0xfff40270      mov.L #IMM:24(#0x646c72), [r5], 
+|           0xfff40275      add   #IMM:4(#0x4), r5, 
+|           0xfff40277      mov.L r10, r1, 
+|           0xfff40279      bsr.A  #0xfffecf,                          ; sym._text_to_morse
+|           0xfff4027d      mov   #IMM:32(#0xfff451f8), r3, 
+|           0xfff40283      mov   #IMM:8(#0x2c), r4, 
+|           0xfff40286      add   #IMM:8(#0xc), r10, r5
+|           0xfff40289      mov.L r5, r1, 
+|           0xfff4028b      mov.L r3, r2, 
+|           0xfff4028d      mov.L r4, r3, 
+|           0xfff4028f      smovf 
+|           0xfff40291      add   #IMM:8(#0xc), r10, r5
+|           0xfff40294      mov.L r5, r1, 
+|           0xfff40296      bsr.A  #0xffff4a,                          ; sym._morse_to_text
+|           0xfff4029a      mov   #IMM:4(#0x0), r5, 
+|           0xfff4029c      mov.L r5, r1, 
+\           0xfff4029e      rtsd  #IMM:8(#0xf), r10, 
+            ; CALL XREF from sym._main @ 0xfff40279
+/ sym._text_to_morse();
+|           0xfff40148      push.L r10, 
+|           0xfff4014a      add   #IMM:8(#0xfc), r0, r10
+|           0xfff4014d      add   #IMM:8(#0xf8), r10, r0
+|           0xfff40150      mov.L r1, [r10], 
+|       ,=< 0xfff40152      bra.B  #0x7a, 
+|      .--> 0xfff40154      mov.L [r10], r5, 
+|      :|   0xfff40156      mov.B [r5], r5, 
+|      :|   0xfff40158      movu.B r5, r5, 
+|      :|   0xfff4015a      cmp   #IMM:8(#0x60), r5, 
+|     ,===< 0xfff4015d      bleu.B  #0x2d, 
+|     |:|   0xfff4015f      mov.L [r10], r5, 
+|     |:|   0xfff40161      mov.B [r5], r5, 
+|     |:|   0xfff40163      movu.B r5, r5, 
+|     |:|   0xfff40165      cmp   #IMM:8(#0x7a), r5, 
+|    ,====< 0xfff40168      bgtu.B  #0x22, 
+|    ||:|   0xfff4016a      mov.L [r10], r5, 
+|    ||:|   0xfff4016c      mov.B [r5], r5, 
+|    ||:|   0xfff4016e      movu.B r5, r5, 
+|    ||:|   0xfff40170      add   #IMM:8(#0x9f), r5, r5
+|    ||:|   0xfff40173      mov   #IMM:32(#0x10), r4, 
+|    ||:|   0xfff40179      mov.L [r5,r4], r5, 
+|    ||:|   0xfff4017c      mov.L r5, dsp:5(#0x1)[r0], 
+|    ||:|   0xfff4017e      mov.L #IMM:32(#loc..LC26], 
+|    ||:|   0xfff40184      bsr.A  #0x14d,                             ; sym._printf ; int printf(const char *format)
+|   ,=====< 0xfff40188      bra.B  #0x3e, 
+|   |``---> 0xfff4018a      mov.L [r10], r5, 
+|   |  :|   0xfff4018c      mov.B [r5], r5, 
+|   |  :|   0xfff4018e      movu.B r5, r5, 
+|   |  :|   0xfff40190      cmp   #IMM:8(#0x40), r5, 
+|   | ,===< 0xfff40193      bleu.B  #0x2c, 
+|   | |:|   0xfff40195      mov.L [r10], r5, 
+|   | |:|   0xfff40197      mov.B [r5], r5, 
+|   | |:|   0xfff40199      movu.B r5, r5, 
+|   | |:|   0xfff4019b      cmp   #IMM:8(#0x5a), r5, 
+|   |,====< 0xfff4019e      bgtu.B  #0x21, 
+|   |||:|   0xfff401a0      mov.L [r10], r5, 
+|   |||:|   0xfff401a2      mov.B [r5], r5, 
+|   |||:|   0xfff401a4      movu.B r5, r5, 
+|   |||:|   0xfff401a6      add   #IMM:8(#0xbf), r5, r5
```

Comment:
I am not sure about this syntax, though; we tend to simplify this stuff. @wargio @Rot127  @ret2libc @thestr4ng3r @imbillow, do you have any opinion on that?

Reply:
or keep the same format with rx-objdump ?

Reply:
let's use something that makes sense like `add 0xbf, r5, r5`

Reply:
It this the syntax of rx-objdump/ISA? Generally I vote for more similarity with the standard tooling. Makes it easier for people to transfer their knowledge/habits when using Rizin.
And if we do simplifications, they should be optional.
Deviating too much from the ISA syntax might makes more trouble in the future.

Reply:
We already diverge from objdump in many other architectures in favor of readability. Thus I am for easier representation.

Reply:
Simplifications are definitely fine, though would prefer to have them in a way, that we can make them optional easily.
Meaning, it should be only a matter of one or two additional if-clauses to go back to standard ISA syntax.

Reply:
well, I will change it to simple syntax likes `add num, r5, r5`